gravitee-apim-console-webui/src/management/api/component/health-check-v4-form/api-health-check-v4-form.component.scss

```
@@ -0,0 +1,46 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+@use '../../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-legacy-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-margin-container;
+}
+
+.healthCheckV4 {
+  &__enableField {
+    width: 100%;
+    padding-bottom: 24px;
+  }
+
+  &__field {
+    width: 100%;
+    padding-bottom: 0px;
```

```suggestion
    padding-bottom: 0;
```
We can do it on next 0px too I think